src/PJ_isea.c

```diff
@@ -59,13 +60,19 @@ int hex_iso(struct hex *h) {
 }
 
 ISEA_STATIC
-int hexbin2(double width, double x, double y,
-                int *i, int *j) {
+void hexbin2(double width, double x, double y, int *i, int *j) {
     double z, rx, ry, rz;
     double abs_dx, abs_dy, abs_dz;
     int ix, iy, iz, s;
     struct hex h;
 
+    if (pj_is_nan(x) || pj_is_nan(y) || pj_is_nan(width)) {
+        fprintf(stderr, "hexbin2 cannot handle NaN inputs\n");
```

Comment:
`proj_log_error` or `proj_log_debug` depending on what happens with the final coordinate

Reply:
Will do.  Lower down in this file, it uses fprintf stderr.  I'll convert that too.

Reply:
Thanks. Looking at this code I think there are several things that can be improved in the future. Especially keeping #423 in mind. This will be a nice start.